I'm sorry, but an uncaught exception occurred.

Compiling ATL code at game/script.rpy:20
  File "game/script.rpy", line 45, in script
    M "OK! Souhaite-moi bonne chance!"
Exception: Expression '5' is not an ATL transform, and so cannot be included in an ATL interpolation.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 45, in script
    M "OK! Souhaite-moi bonne chance!"
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\ast.py", line 2557, in execute
    Say.execute(self)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\ast.py", line 615, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\exports.py", line 1493, in say
    who(what, *args, **kwargs)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\character.py", line 1391, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\character.py", line 1042, in do_display
    display_say(who,
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\character.py", line 738, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\display\core.py", line 2165, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\display\core.py", line 2683, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\display\screen.py", line 485, in visit_all
    self.child.visit_all(callback, seen=None)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\display\displayable.py", line 421, in visit_all
    for d in self.visit():
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\atl.py", line 764, in visit
    block = self.compile()
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\atl.py", line 671, in compile
    block = self.atl.compile(self.context)
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\atl.py", line 873, in compile
    statements = [ i.compile(ctx) for i in self.statements ]
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\atl.py", line 873, in <listcomp>
    statements = [ i.compile(ctx) for i in self.statements ]
  File "C:\Users\game\Desktop\renpy-8.2.0-sdk\renpy\atl.py", line 1225, in compile
    raise Exception("Expression %r is not an ATL transform, and so cannot be included in an ATL interpolation." % expr)
Exception: Expression '5' is not an ATL transform, and so cannot be included in an ATL interpolation.

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.0.24012702
visual novel 1.0
Tue Feb  6 13:21:34 2024
